.gitignore

```
@@ -2,3 +2,4 @@
 /build
 /web/frontend/assets
 /web/frontend/node_modules
+/helm/trento-server/charts/*.tgz
```

you forgot to change the path here as well :)
```suggestion
/packaging/helm/trento-server/charts/*.tgz
```